src/Store/ExtensionClientStoreConnector.re

```diff
@@ -218,15 +216,12 @@ let start = (extensions, setup: Core.Setup.t) => {
       Protocol.BasicProvider.("exthost." ++ string_of_int(provider.id));
     let definitionProvider =
       ExtensionDefinitionProvider.create(client, provider);
+
     dispatch(
-      Oni_Model.Actions.LanguageFeature(
-        Model.LanguageFeatures.DefinitionProviderAvailable(
-          id,
-          definitionProvider,
-        ),
+      Actions.LanguageFeature(
+        LanguageFeatures.DefinitionProviderAvailable(id, definitionProvider),
       ),
     );
-    Log.infof(m => m("Registered suggest provider with ID: %n", provider.id));
```

Comment:
Removed because the action dispatched just before includes exactly the same information